state-cleanup/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use near_sdk::base64;
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::serde_json;
+use near_sdk::{env, near_bindgen};
+
+#[allow(dead_code)]
+#[near_bindgen]
+struct Contract {}
+
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "near_sdk::serde")]
+struct Args {
+    pub keys: Vec<String>,
+}
+
+#[no_mangle]
+pub extern "C" fn clean() {
```

Comment:
There is no guarantee that this can finish within gas limit?

Reply:
I think there is no guarantee here, it is necessary to first calculate the total size of the deleted data before sending the array of keys